File: Arreglos/Java/EliminacionAlFinal.java
package Arreglos.Java;
public class EliminacionAlFinal {
    public static void main(String[] args) {
        int[] inputArr = {11, 21, 31, 41, 51, 61};

        System.out.println("Antes de la eliminación, el array es:");
        for (int j = 0; j < inputArr.length; j++) {
            System.out.print(inputArr[j] + " ");
        }

        int[] nuevoArr = new int[5];
        for (int j = 0; j < nuevoArr.length; j++){
            nuevoArr[j] = inputArr[j];

        }

        System.out.println("\nDespués de la eliminación, el array es: ");
        for (int j = 0; j < nuevoArr.length; j++) {
            System.out.print(nuevoArr[j] + " ");
        }
    }
}

File: Arreglos/C#/Eliminacion al final.cs
using System;
using System.Globalization;
class program
{
    static void Main(string [] args)
    {
        int [] arr = {11, 21, 31, 41, 51, 61};
        
        Console.WriteLine("El array antes de la eliminación es: ");
        foreach (int elemento in arr)
        {
            Console.Write(elemento + " ");
        }
        Console.WriteLine();

        int [] nuevoArr = new int [arr.Length - 1];

        for(int i = 0; i < nuevoArr.Length; i++)
        {
            nuevoArr[i] = arr[i];
        }

        Console.WriteLine("El array después de la eliminación es: ");
        foreach (int elemento in nuevoArr)
        {
            Console.Write(elemento + " ");
        }

    }
}